src/sentry/api/endpoints/project_group_index.py

```diff
@@ -455,6 +454,10 @@ def put(self, request, project):
                             group=group,
                             reason=GroupSubscriptionReason.status_change,
                         )
+                        if not GroupAssignee.objects.filter(
+                            group=group, project=group.project
+                        ).first():
+                            result['assignedTo'] = (User.objects.filter(id=acting_user.id).first())
```

Comment:
We've already got the `active_user` object in memory. Why is this query being performed? (…and why is this expression in parentheses?)